index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Weather Project</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6 offset-3">
+          <header>
+            <h1>Weather Project</h1>
+            <hr />
+          </header>
+          <form>
+            <div class="form-row">
+              <div class="col-md-10">
+                <input
+                  type="text"
+                  class="form-control"
+                  id="city-name"
+                  placeholder="Enter City Name Here"
+                />
+              </div>
+              <button
+                id="search"
+                type="button"
+                class="btn btn-primary col-md-2"
+              >
+                Search
+              </button>
+            </div>
+          </form>
+          <div id="current-weather-conditions" class="row">
+          </div>
+        </div>
+      </div>
+      <div class="row forecast-row">
+    </div>
+
+    <script id="current-weather-template" type="text/x-handlebars">
+      <div class="col-md-6">
+        <p id="current-temp" class="text-center current">{{currentTemp}}&#176;</p>
+        <p id="current-city" class="text-center current">{{currentCity}}</p>
+        <p id="current-condition" class="text-center current">{{currentCondition}}</p>
+      </div>
+      <div id="current-weather-icon" class="col-md-6">
+        <p class="text-center">
+          <img
+            src="http://openweathermap.org/img/wn/{{currentIcon}}@2x.png"
+            alt="weather icon"
+          />
+        </p>
+      </div>
+    </script>
+
+    <script id="forecast-template" type="text/x-handlebars">
+      <div id="day-1" class="col-md-2 offset-md-1 forecast">
+          <p class="text-center forecast-condition">{{day1Condition}}</p>
+          <p class="text-center forecast-temp">{{day1Temp}}&#176;</p>
+          <p class="text-center forecast-icon">
+            <img
+              src="{{day1Icon}}"
+              alt="weather"
+            />
+          </p>
+          <p class="text-center forecast-day">{{day1}}</p>
+        </div>
+
+        <div id="day-2" class="col-md-2 forecast">
```

Comment:
I think you could have created all of these forecast div's dynamically instead of hard coding them